shell/app/modules/project/common/components/issue/edit-issue-drawer.scss

```diff
@@ -0,0 +1,3 @@
+.issue-drawer-tabs .ant-tabs-nav {
+    padding: 0 20px;
```

Comment:
Incorrect indent. style-lint not work?

Reply:
There's something wrong with the style-lint and it doesn't work.